src/main/resources/script_templates/Examples/Blur_Image.scala

```diff
@@ -0,0 +1,26 @@
+// @Dataset inputImg
+// @OpService ops
+// @OUTPUT Dataset(label="Blurred") blurredImg
+
+// A simple Scala script that blurs an input image with
+// a Gaussian filter. Note that you must have an image
+// open to run this script.
+// It is the duty of the scripting framework to bind
+// the input image and then display the blurred output
+// image. It also binds the ImageJ OpService for filtering.
+
+import net.imagej.ops.OpService
+import net.imglib2.img.Img
+import net.imglib2.`type`.numeric.RealType
+
+// Scala is a compiled language with a very strict/powerful
+// type system. We need to cast any parameters that are bound
```

Comment:
Isn't this a limitation for all statically typed languages since the ScriptEngine bindings are typically stored in Map<String, Object>. Do you know whether you can avoid casting for the Kotlin and Java ScriptEngines? Definitely, would be great to avoid having to cast if possible.